.github/workflows/team-channels.json

```diff
@@ -3,7 +3,7 @@
     "Consensus": "eng-consensus",
     "cross-chain-team": "eng-cross-chain",
     "dept-crypto-library": "eng-crypto-mr",
```

Comment:
This one is also `dept-Crypto Library`

Reply:
Oh darn, I guess I missed that. Fixing ...